python/src/trezorlib/definitions.py

```
@@ -115,6 +122,70 @@ def get_eth_token(self, chain_id: int, address: t.AnyStr) -> t.Optional[bytes]:
+                LOG.warning(
+                    "No display format found for address %s \
+                    function signature %s on chain-id %s",
+                    address_str,
+                    func_sig_str,
+                    chain_id,
+                )
+            return result
+
+        # Enumerable backend: drill down chain-id -> address -> function sig.
+        if not entries:
+            LOG.warning(
```

Ruff (0.15.21)

[error] Too many arguments for `logging` format string

(PLE1205)